CMakeLists.txt

```diff
@@ -86,6 +86,13 @@ if(Omega_h_VERSION VERSION_LESS 10.8.0)
     message(FATAL_ERROR "Omega_h version >= 10.8.0 required.")
 endif()
 
+if(EnGPar_VERSION VERSION_LESS 1.1.0)
+    message(FATAL_ERROR "EnGPar version >= 1.1.0 required.")
+endif()
+
+option(PP_PRINT_ENABLED "PUMIPIC print statements enabled" ON)
```

Comment:
Are all of the pumipic variables using the `PP` prefix? It would be better if the variables were `PUMIPIC` prefixed instead. `PP` is much more likely to be a collision.

Reply:
some of them have that prefix, I changed it to PUMIPIC